app/views/content_publisher_feedback_requests/_request_details.html.erb

```diff
@@ -0,0 +1,16 @@
+<div class="alert alert-info">
+  <p>Content Publisher is in private beta and supports most of the features that are available on Whitehall.</p>
+  <p>With this form you can tell us more about your experience with the new app so we can make it better. If you’re having issues publishing or editing a document raise a <a class="govuk-link govuk-link--no-visited-state" href="<%= Plek.new.external_url_for("support") + "/technical_fault_report/new" %>">support request</a> so you can get technical help.</p>
```

Comment:
As this link is displayed within support we don't need to use plek here as you are already on that hostname.

We should also link to this via route rather than hard coding the URL.

Also "govuk-link govuk-link--no-visited-state" has no effect in this application as [govuk-frontend](https://github.com/alphagov/govuk-frontend) is not part of this app.

Reply:
Done. govuk-link govuk-link--no-visited-state has been removed.  The href part now uses a rails path: href="<%= new_technical_fault_report_path %>"> and it works.